### PR TITLE
{chem,data,math}[foss/2020a,intel/2020a] molmod v1.4.5, yaff v1.6.0, h5py v2.10.0 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
I had to disable the tests in the `yaff` easyconfig using `intel/2020a` because of the bug in Intel MPI 2019 update 7 discussed in https://github.com/easybuilders/easybuild-easyconfigs/issues/10213: if a command triggers an `MPI_Init`, it crashes hard in case it wasn't started with an MPI starter like `mpirun` or `srun`, whith an error like:

```
Error in system call pthread_mutex_destroy: Device or resource busy
    ../../src/mpi/init/init_thread_cs.c:60
Abort(2664079) on node 0 (rank 0 in comm 0): Fatal error in PMPI_Init_thread: Other MPI error, error stack:
MPIR_Init_thread(136)........: 
MPID_Init(904)...............: 
MPIDI_OFI_mpi_init_hook(1421): 
MPIDU_bc_table_create(311)...: 
```

The `MPI_Init` gets triggered via `h5py` (which in turns is actually caused by `mpi4py`) in the case of `yaff` (we also prefixed the `import h5py` test with `mpirun` for the same reason).

In case of the `import yaff` test, we can fix the crash by using `mpirun` (via the `%(mpi_cmd_prefix)s` template).

But this doesn't work for the tests, because they in turn fork of subprocesses (not using `mpirun` or equivalent), so the tests still fail...

**Comment by boegel** [comment]
Note to self: sometimes the `import h5py` test fails with `Segmentation fault` or `Floating point exception` when running in a Slurm job, which seems to be fixable by using `export I_MPI_HYDRA_TOPOLIB=ipl` first (see also https://github.com/easybuilders/easybuild-easyconfigs/issues/10314).
It doesn't always occur though, it seems to be related to the number of available cores/amount of available memory in the job?!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, x86_64, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/74c23504f6ebf6d69ce80a606bd9c814 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node3401.kirlia.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/ca941afed84a50069b35f76ad63e3503 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node3149.skitty.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/a780e7f683792c203b455cba1e651c71 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2697.swalot.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/45c465378f7ead36be66b7b7d87da978 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 10 out of 10 (6 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.7.1908, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/4d26bfc99e828ebb4b8f3ce69a5a9340 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/691531af9900960f5d92337650c29ff1 for a full test report.

File: easybuild/easyconfigs/y/yaff/yaff-1.6.0-foss-2020a-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Updated from previous config
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+#
+# Building this in interactive session will result in freeze during either
```
Comment (akesandgren) [inline easybuild/easyconfigs/y/yaff/yaff-1.6.0-foss-2020a-Python-3.8.2.eb:5]: Not true, works without a problem in interactive session

Comment (akesandgren) [reply easybuild/easyconfigs/y/yaff/yaff-1.6.0-foss-2020a-Python-3.8.2.eb:5]: Unless this refers to an interactive Slurm job session, if so it should be made clearer.

Comment (boegel) [reply easybuild/easyconfigs/y/yaff/yaff-1.6.0-foss-2020a-Python-3.8.2.eb:5]: It does refer to the latter, I'll clarify the comment.

**Comment by akesandgren** [comment]
Going in, thanks @boegel!

**Pull request merged**